src/helpers.ts

```diff
@@ -62,7 +62,8 @@ export const convertQuoteToRoute = (step: LiFiStep): Route => {
 
 export const fetchTxErrorDetails = async (txHash: string, chainId: number) => {
   const response = await request<TenderlyResponse>(
-    `https://api.tenderly.co/api/v1/public-contract/${chainId}/tx/${txHash}`
+    `https://api.tenderly.co/api/v1/public-contract/${chainId}/tx/${txHash}`,
+    { disableLiFiErrorCodes: true }
```

Comment:
We use the request function in a couple of places to make requests from sources outside of the LiFi domain - This flag opts out of using the LiFi error codes in situations like this.